addons/mrp/models/mrp_production.py

```diff
@@ -223,6 +223,19 @@ def _get_default_date_planned_finished(self):
     delivery_count = fields.Integer(string='Delivery Orders', compute='_compute_picking_ids')
     confirm_cancel = fields.Boolean(compute='_compute_confirm_cancel')
 
+    mto_mrp_production_child_count = fields.Integer("Number of generated MO", compute='_compute_mto_mrp_production_child_count', store=True)
+    mto_mrp_production_source_count = fields.Integer('Number of source MO', compute='_compute_mto_mrp_production_source_count', store=True)
+
+    @api.depends('procurement_group_id.stock_move_ids.created_production_id')
+    def _compute_mto_mrp_production_child_count(self):
+        for production in self:
+            production.mto_mrp_production_child_count = len(production.procurement_group_id.stock_move_ids.created_production_id)
```

Comment:
https://github.com/odoo/odoo/pull/43366#issuecomment-575114872
len(production.procurement_group_id.stock_move_ids.created_production_id) -> len(production.procurement_group_id.stock_move_ids.mapped('created_production_id.ids'))

Reply:
Mapped is implicit in master for one2many

Reply:
Oh, you are right since 63bc6013ab30acfee7fdedcd396ad3984543b99e it is work